lib/stripe.rb

```diff
@@ -52,6 +52,7 @@
 require 'stripe/invoice_item'
 require 'stripe/order'
 require 'stripe/order_return'
+require 'stripe/payout'
 require 'stripe/plan'
 require 'stripe/product'
 require 'stripe/recipient'
```

Comment:
@remi-stripe Haha, sorry for the churn, but one more: you'll need a require on `recipient_transfer` here. The test suite would have caught this if it wasn't so broken right now.